source/server/overload_manager_impl.cc

```diff
@@ -152,6 +152,14 @@ void OverloadManagerImpl::start() {
   timer_->enableTimer(refresh_interval_);
 }
 
+void OverloadManagerImpl::stop() {
```

Comment:
Is it easy to write a direct test for this?

Reply:
Added a simple unit test to check that the timer is disabled when stopping.